### PR TITLE
Editing workflows

### LINKED ISSUE #1103: vTHL3: Edit workflow
via administration ui

- [x] rename
- [x] change handlers

(split out from #867)

### PR DESCRIPTION
**Pull request by opqdonut**
- reuse create-workflow functionality for editing
- edit button when viewing workflow

closes #1103

### CODE REVIEW

File: src/cljs/rems/administration/create_workflow.cljs [hunk 1]
```diff
@@ -9,54 +10,99 @@
...
  ::enter-page
- (fn [{:keys [db]}]
-   {:db (assoc db
-               ::form {:type :dynamic}
-               ::loading? true)
-    ::fetch-actors nil}))
-
-(rf/reg-sub ::loading? (fn [db _] (::loading? db)))
+ (fn [{:keys [db]} [_ workflow-id]]
+   (let [parsed-id (parse-int workflow-id)]
```
Comment (Macroz) [inline src/cljs/rems/administration/create_workflow.cljs]: Maybe the parsing could be already in `spa.cljs` and this function could be purer.

File: test/cljs/rems/test_util.cljs [hunk 2]
```diff
@@ -17,3 +18,9 @@
   (is (= nil (localize-time "")))
   (is (= (expected-time test-time) (localize-time #inst "1980-01-02T13:45:00.000Z")))
   (is (= nil (localize-time nil))))
+
+(deftest test-parse-int
+  (is (= nil (parse-int nil)))
+  (is (= nil (parse-int "")))
+  (is (= nil (parse-int "")))
```
Comment (Macroz) [inline test/cljs/rems/test_util.cljs]: Something wrong here?

Comment (opqdonut) [reply test/cljs/rems/test_util.cljs]: should've been `"a"`

File: src/cljs/rems/administration/create_workflow.cljs (hunk 1 above)
Comment (opqdonut) [reply src/cljs/rems/administration/create_workflow.cljs]: good idea, done

File: src/clj/rems/api/workflows.clj [hunk 3]
```diff
@@ -2,12 +2,13 @@
   (:require [clj-time.core :as time-core]
             [clojure.test :refer [deftest is]]
             [compojure.api.sweet :refer :all]
-            [rems.api.applications :refer [User get-users]]
+            [rems.api.applications :refer [User get-users format-user]]
```
[the remark this answers is not in the document]
Comment (opqdonut) [reply src/clj/rems/api/workflows.clj]: this is backend code

**Pull request merged**